cypress/e2e/models/migration/controls/stakeholders.ts

```diff
@@ -59,7 +59,16 @@ export class Stakeholders {
         if (groups) this.groups = groups;
     }
 
-    public static openList(itemsPerPage = 100): void {
+    public static openList(forceReload = false): void {
+        const itemsPerPage = 100;
```

Comment:
If you remove the `itemsPerPage` parameter from the method you can remove the variable as well.

Its clearer to just have `selectItemsPerPage(100);`